includes/class-img-converter.php

```
@@ -904,20 +904,17 @@ public static function get_img_path( string $source_image, string $format = 'web
 				if ( empty( $local_path ) ) {
 					// If Util::get_local_path failed, manually resolve if it's a URL.
 					static $home_url    = array();
-					static $content_url = array();
+					$content_url_base   = untrailingslashit( Util::cached_content_url( '' ) );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Util' in method 'get_img_path'. (undefined)

(StaticAccess)